gallery/templates/nav.html

```diff
@@ -61,4 +61,11 @@
             </div>
         </div>
     </nav>
+    <div class="text-center" id="plug_integration"></div>
+    
+    <script>
+        if (window.location.pathname == '/view/dir/1'){
+        	document.getElementById("plug_integration").innerHTML = '<a href="https://plug.csh.rit.edu"><img src="https://plug.csh.rit.edu/data" alt="Advertisements By CSH Plug!"></a>';
+        }
+    </script>
```

Comment:
Please don't do this this way... Put stuff in the `view_dir` template. You can check `directory.id` in that file and only insert the plug if it's in dir 1. (see: `{% if ... %}` directives throughout that file)

Reply:
Want to talk later tonight? Still trying to get the hang of this and a little confused lol

Reply:
Sure :call_me_hand: